assets/xml/overlays/ovl_En_Ganon_Organ.xml

```diff
@@ -0,0 +1,33 @@
+<Root>
+    <File Name="ovl_En_Ganon_Organ" BaseAddress="0x80A28000" RangeStart="0x3E0" RangeEnd="0x6F70" Segment="128">
+        <!-- D_80A283E8 -->
```

Comment:
all the `D_` comments can go I think

Reply:
I agree, but this is also apparently present in other files (xmls for overlays specifically). Anyone know why? Dont think this information is important to keep